editor/state.js

```diff
@@ -366,17 +309,15 @@ export function isTyping( state = false, action ) {
 }
 
 /**
- * Reducer returning multi selected block state.
+ * Reducer returning the block selection's state.
  *
  * @param  {Object} state  Current state
  * @param  {Object} action Dispatched action
  * @return {Object}        Updated state
  */
-export function multiSelectedBlocks( state = { start: null, end: null }, action ) {
+export function blockSelection( state = { start: null, end: null }, action ) {
```

Comment:
Not sure I understand the note. Do you mean adding a comment to explain the initial state?